<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CatScanDialog</class>
 <widget class="QDialog" name="CatScanDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>227</width>
    <height>329</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>配置扫描端口</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
     <item>
      <widget class="QSpinBox" name="portBox">
       <property name="maximum">
        <number>9999999</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="addBtn">
       <property name="text">
        <string>添加</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListWidget" name="portList"/>
   </item>
   <item>
    <widget class="QToolButton" name="rmBtn">
     <property name="text">
      <string>移除</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>完成等待时间</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="waitBox">
       <property name="maximum">
        <number>100000</number>
       </property>
       <property name="singleStep">
        <number>100</number>
       </property>
       <property name="value">
        <number>3000</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CatScanDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>113</x>
     <y>308</y>
    </hint>
    <hint type="destinationlabel">
     <x>113</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CatScanDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>113</x>
     <y>308</y>
    </hint>
    <hint type="destinationlabel">
     <x>113</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
